_datascience/using_workflows.md

```diff
@@ -1,39 +1,235 @@
+
+### Nextflow
+
+- **[Nextflow Training](https://training.nextflow.io/)** - Official Nextflow training
+- **[nf-core Usage Docs](https://nf-co.re/usage/introduction)** - Running nf-core pipelines
+- **[Nextflow Patterns](https://nextflow-io.github.io/patterns/)** - Common workflow patterns
+
+## Best Practices
+
+Regardless of which workflow system you choose:
+
+1. **Start small** - Test with small datasets before scaling up
+2. **Use version control** - Track your workflows in Git
+3. **Containerize** - Use Docker/Apptainer for reproducibility
+4. **Document** - Include README files and example inputs
+5. **Test** - Validate outputs before running large analyses
+6. **Share** - Contribute back to community libraries
+7. **Version workflows** - Tag releases when publishing results
 
-When performing bioinformatic analysis, scientists often need to perform a series of interconnected
-computational transformations of the raw input data. While it is possible to coordinate multiple tasks
-using BASH scripts or batch submission to a SLURM cluster, it can be far more convenient to use
-software which has been specifically designed to coordinate these types of workflows. Some examples
-of those workflow managers are:
+## Advanced Topics
 
-[Cromwell/WDL](https://cromwell.readthedocs.io/en/stable/), 
-and [Nextflow](https://nextflow.io/).
+Once comfortable with basic workflows:
 
+- **Cloud computing** - Scale to AWS Batch or other cloud platforms
+- **Custom containers** - Build your own Docker images with [WILDS Docker Library](https://github.com/getwilds/wilds-docker-library)
+- **Workflow optimization** - Fine-tune resource requests and parallelization
+- **CI/CD** - Automate workflow testing with GitHub Actions
+- **Publishing** - Share workflows via GitHub for reproducible research
 
-## Getting Started with WDL
+## Common Questions
 
-You can find a wealth of information on getting started using WDL workflows at Fred Hutch on the [Resource Library entry for WDL](/compdemos/Cromwell/) and on our [PROOF page](/datascience/proof/).  PROOF is a tool that wraps the Cromwell workflow manager for use at Fred Hutch.  
+**Q: Can I convert between WDL and Nextflow?**
+A: Not directly. While both accomplish similar goals, their syntaxes are fundamentally different. Choose based on your needs and stick with it for a given project.
 
+**Q: Which is faster?**
+A: Performance depends more on your workflow design and resource allocation than the workflow system itself. Both are highly optimized.
 
+**Q: Can I run WDL workflows without PROOF?**
+A: Yes! You can use Sprocket, miniWDL, or Cromwell directly. See the [WDL Execution Engines guide](/datascience/wdl_execution_engines/).
 
-## Getting Started with Nextflow
+**Q: Do I need to learn Docker?**
+A: Basic Docker knowledge is helpful but not required. Both WILDS WDL Library and nf-core provide pre-built containers. Workflow engines handle container pulling automatically.
```

Comment:
```suggestion
A: Basic Docker knowledge is very helpful. Pre-built Docker containers exist for both WDL and Nextflow and the workflow engines handle container pulling automatically. So while you may not need to build your own container, you are likely to be working with containers.
```
This has been everyone's biggest sticking point when getting started with WDL (including DaSL folks!) so I would just say yes to mentally prepare them